src/services/osidb-errors-helpers.ts

```diff
@@ -0,0 +1,52 @@
+
+
+export function getDisplayedOsidbError(error: any): string {
+  if (error.response) {
+    if (error.response.headers?.['content-type']?.startsWith('text/html')) { // OSIDB Server might be in debug mode?
+      return parseOsidbHtmlError(error);
```

Comment:
This case is not being tested

```js
it('should handle stack trace errors', () => {
  const error = {
    response: {
      headers: {
        'content-type': 'text/html'
      },
      data: '<div class="exception_value">Some error on line 1</div>'
    },
  };

  expect(getDisplayedOsidbError(error)).toBe(
    'Likely error between OSIDB and database:\nSome error on line 1'
  );
});
```

Reply:
Can you clarify what you mean? The test does exactly what it should. In any case, I think I will alter the test case language to be more descriptive.

Reply:
Sorry I meant in the case of a error with `text/html` `content-type`, the call to the `parseOsidbHtmlError` si not texted, the code snippet was my suggestion for the test

Reply:
Oh yes, we should add this test case!

Reply:
Sorry I thought the test case that **you added here** was already a part of the test suite in the PR 🫠. Thank you for the test case